starlette_admin/views.py

```diff
@@ -750,11 +750,14 @@ async def serialize(
                 "result": await self.select2_result(obj, request),
             }
         obj_meta["repr"] = await self.repr(obj, request)
+
+        # Make sure the primary key is always available
+        pk_attr = not_none(self.pk_attr)
+        if pk_attr not in obj_serialized:
+            pk_value = await self.get_serialized_pk_value(request, obj, action)
```

Comment:
We need to use `get_serialized_pk_value` at these lines as well.
https://github.com/jowilf/starlette-admin/blob/8c6f58e27c76649a78feb92abd5efcdd56636d0e/starlette_admin/views.py#L722

https://github.com/jowilf/starlette-admin/blob/8c6f58e27c76649a78feb92abd5efcdd56636d0e/starlette_admin/views.py#L732